rust-runtime/aws-smithy-http-server-python/stubgen.py

```diff
@@ -401,6 +386,19 @@ def walk_module(writer: Writer, mod: Any):
             print(f"Unknown type: {member}")
```

Comment:
With the new `CODEGEN_VERSION` constant this script will fail here with: `Unknown type: 0.0.0-smithy-rs-head`.

I think we can add support for scalars here:

```
elif is_scalar(member):
    writer.define(f"{name}: {type(member).__name__} = ...")
else:
    print(f"Unknown type: {member}")

...

def is_scalar(obj: Any) -> bool:
    return isinstance(obj, (str, float, int, bool))
```